project_event/wizard/project_event_wizard.py

```diff
@@ -0,0 +1,220 @@
+            'res_model': self._name,
+            'views': [(view.id, 'form')],
+            'target': 'new',
+            'res_id': self.id,
+            'context': self.env.context,
+        }
+
+    @api.multi
+    def add_tasks(self):
+        self.ensure_one()
+        activities = self.activity_line_ids
+        view = self.env.ref(
+            'project_event.view_project_task_wizard')
+        for act in activities:
+            for task in act.template_id.task_template_ids:
+                task_vals = {
+                    'event_wiz_id': self.id,
+                    'template_id': task.id,
+                    'activity_wiz_id': act.id,
+                    'task_name': task.name,
+                    'task_resp_id': task.temp_resp_id.id,
+                    'task_category_id': task.task_category_id.id,
+                    'resource_id': task.resource_id.id,
+                    'department_id': task.department_id.id,
+                    'duration': task.duration,
+                    'start_time': task.start_time,
+                    'notes': task.notes,
+                }
+                self.env['task.template.line.wiz'].create(task_vals)
+
+        return {
+            'name': _('Tasks Wizard'),
+            'type': 'ir.actions.act_window',
+            'view_type': 'form',
+            'view_mode': 'form',
+            'res_model': self._name,
+            'views': [(view.id, 'form')],
+            'target': 'new',
+            'res_id': self.id,
+            'context': self.env.context,
+        }
+
+    @api.multi
+    def create_event_from_template(self):
+        self.ensure_one()
+        event_vals = {
+            'name': self.name,
+            'responsible_id': self.event_resp_id.id,
+            'notes': self.event_notes,
+            'project_type': 'event',
+        }
+        event = self.env['project.project'].create(event_vals)
+        for act in self.activity_line_ids:
+            tasks = self.env['task.template.line.wiz'].search([
+                ('event_wiz_id', '=', self.id),
+                ('activity_wiz_id', '=', act.id),
+            ])
+            activity_vals = {
+                'name': act.name,
+                'project_id': event.id,
+                'responsible_id': act.activity_resp_id.id,
+                'activity_category_id': act.activity_category_id.id,
+                'activity_task_type': 'activity',
+                # add room_id
+                'notes': act.notes,
+                'child_ids': [
+                    (0,
+                     0,
+                     {
+                         'name': task.task_name,
+                         'activity_task_type': 'task',
+                         'task_responsible_id': task.task_resp_id.id,
+                         'task_category_id': task.task_category_id.id,
+                         'department_id': task.department_id.id,
+                         'notes': task.notes,
+                         # add resource_id
```

Comment:
@rimbendhaou: if not required,  remove the line 123, 135  # add resource_id
Note:- If these fields are going to add in the future, work need new task number.